package.json

```diff
@@ -60,20 +60,20 @@
     "yup": "^0.28.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.8.4",
-    "@babel/preset-env": "^7.8.4",
-    "@babel/preset-react": "^7.8.3",
-    "@babel/preset-typescript": "^7.8.3",
+    "@babel/core": "^7.12.10",
+    "@babel/preset-env": "^7.12.11",
+    "@babel/preset-react": "^7.12.10",
+    "@babel/preset-typescript": "^7.12.7",
     "@graphql-codegen/cli": "1.15.4",
     "@graphql-codegen/typescript": "1.15.4",
     "@graphql-codegen/typescript-operations": "1.15.4",
     "@graphql-codegen/typescript-react-apollo": "1.15.4",
     "@graphql-codegen/typescript-resolvers": "1.15.4",
-    "@storybook/addon-actions": "^5.3.13",
-    "@storybook/addon-storyshots": "^5.3.13",
-    "@storybook/addon-storysource": "^5.3.19",
-    "@storybook/addon-viewport": "^5.3.13",
-    "@storybook/react": "^5.3.13",
+    "@storybook/addon-actions": "^6.1.11",
+    "@storybook/addon-storyshots": "^6.1.11",
+    "@storybook/addon-storysource": "^6.1.11",
+    "@storybook/addon-viewport": "^6.1.11",
+    "@storybook/react": "^6.1.11",
```

Comment:
We are using ^ versions already, so I think it would make more sense to update all of the versions to ^N e.g ^5. Since the minor and patch versions become redundant with ^, it should be clearer that way. Thoughts?